tests/network/macvtap.go

```diff
@@ -75,19 +75,18 @@ var _ = SIGDescribe("Macvtap", decorators.Macvtap, func() {
 			To(Succeed(), "A macvtap network named %s should be provisioned", macvtapNetworkName)
 	})
 
-	newAlpineVMIWithMacvtapNetwork := func(macvtapNetworkName string) *v1.VirtualMachineInstance {
-		return libvmi.NewAlpine(
-			libvmi.WithInterface(
-				*v1.DefaultMacvtapNetworkInterface(macvtapNetworkName)),
-			libvmi.WithNetwork(libvmi.MultusNetwork(macvtapNetworkName, macvtapNetworkName)))
+	newAlpineVMIWithMacvtapNetwork := func(macvtapNetworkName string, opts ...libvmi.Option) *v1.VirtualMachineInstance {
```

Comment:
I do not see the usefulness of these helper functions, they just make a mess.

Can you simply inline the building of the VMI?

(BTW, I do not understand the slice appending done here, it converts the options into a slice which at the end expands them).